src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public int getRandomBall() {
```

Comment:
The method name `getRandomBall` suggests that it should return a 'Ball' object, but it currently returns an `int`. Either the method should return a `Ball` object, or the method name should be changed to reflect that it returns a random number.